Gemfile

```diff
@@ -2,6 +2,10 @@ source 'https://rubygems.org'
 
 gemspec
 
+platforms :ruby_19 do
+  gem 'json', '~> 1.8'
+end
+
```

Comment:
Version of the `json` gem greater than or equal to 2.0.0 are not compatible with Ruby 1.9, causing a build error on the Travis CI. This allows Travis to get past the `bundle` phase and run the tests successfully on Ruby 1.9.